src/fides/api/ctl/database/seed.py

```diff
@@ -89,14 +85,7 @@ def create_or_update_parent_user() -> None:
         )
         FidesUserPermissions.create(
             db=db_session,
-            data={
-                "user_id": user.id,
-                "scopes": [
-                    PRIVACY_REQUEST_CREATE,
-                    PRIVACY_REQUEST_READ,
-                    PRIVACY_REQUEST_TRANSFER,
-                ],
-            },
+            data={"user_id": user.id, "roles": [OWNER]},
```

Comment:
We're no longer giving users scopes directly - in creating this parent user, there is no existing role that has these scopes. Making `owner` instead - does that work when seeding this data @adamsachs?

Reply:
yes, that should be fine. the only risk i'd see is that we're giving the "parent user" broader scope than is strictly necessary, but no human user should actually be authenticating as this user, it's just meant for inter-server communications (i.e. the fides _child_ uses these credentials to connect to _this_ fides, which is effectively acting as a _parent_ if it has these variable set). so as long as we don't update the fides connector code to accidentally do destructive operations on the parent fides it connects to, there's no impact here.

Reply:
thanks for weighing in Adam!